Change buttondown to mailchimp for email signup forms

raq929 opened the pull request:
I had to modify some selectors to be a lot more specific because of how the submit button is now an input not a button. If there are better ways to do it (I feel like there probably are, but a bunch of things I tried did not work), let me know. 

Also, I'm unclear on what needs to be different between the form that is in the footer and the form on the entrepreneur page.

Suggestions for what link/info to put in the README are welcome; I don't feel like I know enough about MailChimp to be very specific.

Commit:
Change buttondown to mailchimp for email signup forms

Commit:
Use an id to get enough specificity on the button

sdzaveri commented:
Could we add some spacing above the field label on the apply page. Looks like there was .25rem before:
<img width="516" alt="Screen Shot 2021-07-26 at 3 54 08 PM" src="https://user-images.githubusercontent.com/3188037/127010402-3f8d88f0-f87f-4533-af3e-3ce0636bd041.png">
<img width="498" alt="Screen Shot 2021-07-26 at 3 53 36 PM" src="https://user-images.githubusercontent.com/3188037/127010411-3442d8e3-0c86-4433-b2b8-c2524205d515.png">

sdzaveri commented:
Clicking the Subscribe button isn't actually working for me in the preview. Also, if possible could we get the success and in progress states? Let me know if we need to design something else or I need to look into mailchimp stuff:

<img width="583" alt="Screen Shot 2021-07-26 at 3 55 42 PM (2)" src="https://user-images.githubusercontent.com/3188037/127010847-d872fccb-3358-4616-b8ec-64c0f05cd5d0.png">
<img width="577" alt="Screen Shot 2021-07-26 at 3 55 29 PM" src="https://user-images.githubusercontent.com/3188037/127010856-5e84172d-ff1e-44d4-bbcf-eb34c0f5b9c5.png">
<img width="513" alt="Screen Shot 2021-07-26 at 3 55 00 PM" src="https://user-images.githubusercontent.com/3188037/127010882-62d7260e-b013-4548-975e-0418938f2845.png">

Commit:
Fix margin on in-page form

Commit:
Remove form js

Allows the default behavior of the submit button, which sends the email to mailchimp

Commit:
Add signup location
